### ISSUE TITLE
Persistent Save does not restore destroyed tank

### ISSUE THREAD

**Issue by zverofaust** [A3Antistasi/antistasiofficial#121]
Long story short, as per Issue #120 my tank was destroyed by a phantom respawning outpost after I retrieved it from garage (BMD APC). However upon restarting and reloading my last save, the BMD was gone from my garage despite the fact I didn't save after retrieving it. 

STR:

Buy BMD
Store in Garage
Persistent Save
Abort & Restart Scenario
Load Previous Save
Retrieve BMD from garage
Destroy it
Abort & Restart Scenario (do not persistent save)
Load Previous Save
BMD Gone

**Comment by StefArma** [comment]
True, i think this happened when we made player gears saved on disconnect... probably also garaged vehicles are bounded in the same way.

It make sense in multiplayer but indeed is a thing in singleplayer where using save and load isn't only a way to continue the war but also to roll back what went wrong.

Will investigate

**Comment by StefArma** [comment]
Referencing Forumpost: http://www.a3antistasi.com/forum/m/38173738/viewthread/31157487-restarting-losing-vehicles-in-singleplayer/page/1

**Comment by zalexki** [comment]
Seems like #285
